zebra/if_netlink.c

```diff
@@ -66,6 +66,7 @@
 #include "zebra/kernel_netlink.h"
 #include "zebra/if_netlink.h"
 
+extern int vrf_with_netns;
 
```

Comment:
Could it be defined into a .h with a proper comment explaining the use of this extern?

Reply:
done.